### DISCUSSION TITLE
想问问这些统计的楼盘有后续么？有没有在解决中的？

### DISCUSSION THREAD

**Discussion by CodeJiang** [WeNeedHome/SummaryOfLoanSuspension#1131] [Q&A]
RT

**Comment by xijinping0** [comment]
等二十大我连任以后，我会亲自解决的！要相信党，相信中央。

----

推荐阅读：[《习近平与他的情人们》](https://xijinping0.github.io/lovers/docs/toc)

**Reply by ghost** [reply]
对捏，我相信这个问题肯定会亲自解决的，解决有问题的人也是解决捏🤣。

**Comment by volans19** [comment]
对于这些群体性事件，党的一贯解决步骤为：

1.先收买一些不坚定者，放弃维权
2.挑拨离间坚定和不坚定者，让他们内部互斗互耗
3.抓出挑事的领头者，往死里打，威胁全家
4.最后用各种社会控制工具，网络封号，辟谣，红码，抓记者，抓律师等等“雷霆行动”，收尾

解决完毕

**Comment by Cegazhos** [comment]
暴雷的恒大 融创这些很难复工了，除非有人(例如当地政府)愿意垫付后续复工费用。
这类没希望的业主应该是要走法律程序 取消交易 少亏一点。


总的来看，烂尾复工交房的 不多

**Comment by xiejun946** [comment]
看区域政府，武汉的话政府有专班，给钱工人施工，就是资金不足，比较慢，但是交付了一部分

**Reply by xiejun946** [reply]
恒大科技旅游城
